src/pages/WorkspaceSwitcherPage/index.tsx

```diff
@@ -158,7 +158,7 @@ function WorkspaceSwitcherPage() {
     return (
         <ScreenWrapper
             testID={WorkspaceSwitcherPage.displayName}
-            includeSafeAreaPaddingBottom={false}
+            includeSafeAreaPaddingBottom
```

Comment:
It looks like we enabled `includeSafeAreaPaddingBottom` here and disabled `includeSafeAreaPaddingBottom` in `SelectionList` below. Why did we do this?

Reply:
@rojiphil I think it caused a jump or was showing an additional space above the keyboard for a fraction of a second. I fixed it in https://github.com/Expensify/App/pull/42143/commits/491c481a0c1d9049615d1f93c9cdb92c6238c3da

And I think it was happening because a wrong component was trying to handle insets

That's the issue that was fixed by these changes: https://github.com/Expensify/App/pull/42143#issuecomment-2368206160